src/Symfony/Bundle/FrameworkBundle/HttpCache/HttpCache.php

```diff
@@ -62,6 +63,15 @@ public function __construct(KernelInterface $kernel, $cache = null, SurrogateInt
         parent::__construct($kernel, $this->createStore(), $this->createSurrogate(), array_merge($this->options, $this->getOptions()));
     }
 
+    public function handle(Request $request, int $type = HttpKernelInterface::MASTER_REQUEST, bool $catch = true)
+    {
+        if ($this->kernel->getContainer()->getParameter('kernel.http_method_override')) {
```

Comment:
`if (($c =$this->kernel->getContainer())->hasParameter('kernel.http_method_override') && $c->getParameter('kernel.http_method_override')) {`

Reply:
nvm, this is in fwb, where the parameter is always defined